src/target/riscv/riscv-013.c

```diff
@@ -1181,9 +1181,11 @@ static int prep_for_register_access(struct target *target,
 {
 	assert(orig_mstatus);
 
-	if (!is_fpu_reg(regno) && !is_vector_reg(regno))
+	if (!is_fpu_reg(regno) && !is_vector_reg(regno)) {
+		*orig_mstatus = 0;
```

Comment:
Please, add a comment, explaining that this assignment is needed to silence the static analyzer warning. Also, if you like, I would suggest to move the initialization to the point of declaration. IMHO, this would make it clearer (if not FPU nor vector register, do nothing in `prep_...()` and in `cleanup_...()`)

Reply:
Done.